src/model/backup.ts

```diff
@@ -168,7 +171,7 @@ export async function restoreCloudBackup(
       // Restore only wallets that were backed up in cloud
       // or wallets that are read-only
       const walletsToRestore: AllRainbowWallets = {};
-      forEach(userData.wallets, wallet => {
+      Object.values(userData.wallets).forEach(wallet => {
```

Comment:
nit: can we just do one more check on `userData.wallets` also in this file

Reply:
sure